examples/basic.js

```diff
@@ -1,10 +1,10 @@
-import React, {Component} from 'react';
-import {render} from 'react-dom';
-import {SortableContainer, SortableElement, arrayMove} from 'react-sortable-hoc';
+import React, { Component } from 'react';
+import { render } from 'react-dom';
```

Comment:
@clauderic yeah we can do that. I have no preference. VS Code is my main editor and I use the shortcut to format code and that was the result. I just have to change my vs code settings somewhere.